### PR TITLE
[ADD] module: purchase_lot

### PR DESCRIPTION
**Pull request by Kev-Roche**
This module propagate lot from sale order to purchase order.
This is a copy of https://github.com/akretion/ak-odoo-incubator/tree/14.0/purchase_lot by @bealdav  and @florian-dacosta (I only change read me and manifest sections). 

cc @sebastienbeau

### CODE REVIEW

**Review by florian-dacosta — APPROVED** [review]

File: purchase_lot/models/purchase_order_line.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
+##############################################################################
+#
+#  licence AGPL version 3 or later
+#  see licence in __openerp__.py or http://www.gnu.org/licenses/agpl-3.0.txt
+#  Copyright (C) 2022 Akretion (http://www.akretion.com).
+#  @author David BEAL <david.beal@akretion.com>
+#
+##############################################################################
```
Comment (HviorForgeFlow) [inline purchase_lot/models/purchase_order_line.py]: Could you adapt this licence header to looks like normal OCA ones?

File: purchase_lot/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Copyright (C) 2022 Akretion (<http://www.akretion.com>).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Purchase Lot",
+    "version": "14.0.1.0.0",
+    "category": "Purchase",
+    "license": "AGPL-3",
+    "author": "Akretion, Odoo Community Association (OCA)",
+    "maintainers": ["florian-dacosta "],
```
Comment (HviorForgeFlow) [inline purchase_lot/__manifest__.py]: Drop last whitespace

**Review by HviorForgeFlow — CHANGES_REQUESTED** [review]
Module looks promising! Thanks for the contribution! 

Code review

File: purchase_lot/models/purchase_order_line.py (hunk 1 above)
Comment (Kev-Roche) [reply purchase_lot/models/purchase_order_line.py]: Sure

File: purchase_lot/__manifest__.py (hunk 2 above)
Comment (Kev-Roche) [reply purchase_lot/__manifest__.py]: My bad, I drop it

**Comment by sebastienbeau** [comment]
It will be better to keep the history of commit.
Inspired from here : https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-14.0


```
git clone https://github.com/OCA/purchase-workflow -b 14.0
git checkout -b 14.0-mig-$MODULE origin/14.0
git remote add ak https://github.com/akretion/ak-odoo-incubator
git format-patch --keep-subject --stdout origin/14.0..ak/14.0 -- $MODULE | git am -3 --keep
```

**Comment by Kev-Roche** [comment]
> keep the history of commit
@sebastienbeau 
You are right, it's done.

**Review by HviorForgeFlow — APPROVED** [review]
Code review LGTM

**Review by HviorForgeFlow — APPROVED** [review]
Functional Review LGTM

**Pull request merged**